papers/Vvmax/calc_vvmax.py

```diff
@@ -0,0 +1,764 @@
+"""
+Script to calculate V and Vmax for CRAFT FRBs
+"""
+
+import numpy as np
+import os
+#from zdm import cosmology as cos
+# sets up cosmological grid
+#    cos.set_cosmology()
+#    cos.init_dist_measures()
+from zdm import pcosmic
+from scipy import interpolate
+from zdm import real_loading as loading
+from matplotlib import pyplot as plt
+from scipy.special import jv as BessJ
+from scipy import constants
+from pkg_resources import resource_filename
+from zdm import cosmology as cos
+from scipy.integrate import quad
+import time
+
+
+# initialises cosmology and distance measures 
+
+
+def main(fname,outfile,Nsfr,alpha=0.,loc=False,minz=None,write=True,
+    laststrings=None,LimZ=None,zfrac = 0.):
+    """
+    
+    """
+    global macquartz
```

Comment:
`global`s are so...  lazy  ;)

Reply:
Yeah I know. Will tell you privately some of this history of this at some point